src/redis_protocol/proxy_lib/redis_parser.cpp

```diff
@@ -369,7 +368,7 @@ void redis_parser::reply_all_ready()
     std::vector<dsn::message_ex *> ready_responses;
     fetch_and_dequeue_messages(ready_responses, true);
     for (dsn::message_ex *m : ready_responses) {
-        dassert(m != nullptr, "");
+        CHECK(m, "");
```

Comment:
It's better to user `m != nullptr` for pointer

Reply:
Both are OK.
But in the future refactor, we are going to introduce `CHECK_EQ` and `CHECK_NE`, it's a bit of duplicate and strange if we judge the pointer like `CHECK_NE(pointer, nullptr)`. I also consulted some other projects' habit, both styles are OK.

Reply:
Added a new macro `CHECK_NOTNULL` for pointers